.github/workflows/ci.yaml

```diff
@@ -22,6 +22,8 @@ jobs:
       - uses: actions/setup-node@master
         with:
           node-version: 16
+          cache: "npm"
+          cache-dependency-path: "package-lock.json"
```

Comment:
We don't need to point to the package-lock.json explicitly when the file is in the root.
[According to the docs](https://github.com/actions/setup-node#caching-global-packages-data)

Reply:
Addressed in ad7ecee0417ffe4da33d578f8e77a62ecd94527e